src/it/java/LmSensorsIT.java

```diff
@@ -0,0 +1,97 @@
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.metricshub.engine.client.ClientsExecutor;
+import org.metricshub.engine.configuration.HostConfiguration;
+import org.metricshub.engine.configuration.IConfiguration;
+import org.metricshub.engine.connector.model.ConnectorStore;
+import org.metricshub.engine.connector.model.common.DeviceKind;
+import org.metricshub.engine.extension.ExtensionManager;
+import org.metricshub.engine.strategy.collect.CollectStrategy;
+import org.metricshub.engine.strategy.collect.PrepareCollectStrategy;
+import org.metricshub.engine.strategy.collect.ProtocolHealthCheckStrategy;
+import org.metricshub.engine.strategy.detection.DetectionStrategy;
+import org.metricshub.engine.strategy.discovery.DiscoveryStrategy;
+import org.metricshub.engine.telemetry.HostProperties;
+import org.metricshub.engine.telemetry.TelemetryManager;
+import org.metricshub.extension.oscommand.OsCommandExtension;
+import org.metricshub.extension.oscommand.SshConfiguration;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+public class LmSensorsIT {
+	static {
+		Locale.setDefault(Locale.US);
+	}
+
+	private static final Path CONNECTOR_DIRECTORY = Paths.get(

```

Comment:
We should load the connectors from `src/main/connector`, it is not required to duplicate them here.